media/css/mozorg/home/home-a.less

```diff
@@ -0,0 +1,1097 @@
+@import '../../pebbles/lib.less';
+
+// Variables and Mixins
+
+@color-mozred-bright: saturate(lighten(@color-mozred, 5%), 10%);
+
+.invert-text() {
+    color: #fff;
+
+    a:link,
+    a:visited,
+    a:hover,
+    a:active,
+    a:focus {
+        color: #fff;
+    }
+}
+
+
+// General styles
+body {
+    border-top: 0;
+}
+
+.section-lead,
+.section-body {
+    padding: 20px;
+}
+
+.section-lead h2 {
+    .font-size(36px);
+    font-style: italic;
+    font-weight: bold;
+    text-align: center;
+}
+
+.section-lead {
+    .font-size(20px);
+}
+
+.section-body {
+    .font-size(20px);
+
+    h3 {
+        .font-size(32px);
+        font-weight: normal;
+    }
+}
+
+.section-cta {
+    .font-size(26px);
+    font-weight: bold;
+    margin: 20px auto;
+    text-align: center;
+
+    a {
+        .trailing-arrow;
+        text-decoration: none;
+        position: relative;
+
+        &:after {
+            .transition(right 150ms ease);
+            display: inline-block;
+            position: relative;
+            right: 0;
+            text-decoration: none;
+        }
+
+        &:hover,
+        &:focus {
+            text-decoration: underline;
+
+            &:after {
+                right: -4px;
+            }
+        }
+    }
+}
+
+// Header
+.masthead {
+    .invert-text;
+    background: @color-mozred;
+
+    .content {
+        padding: 61px 20px 0;
+    }
+
+    .masthead-logo {
+        background: @color-mozred-bright;
+        height: 30px;
+        left: 0;
+        margin: 0;
+        padding: 15px 0;
+        position: absolute;
+        text-align: center;
+        top: 0;
+        width: 100%;
+
+        img {
+            height: 30px;
+            margin-left: 60px;
+        }
+    }
+}
+
+// Global main nav
+.masthead-nav-main {
```

Comment:
I'd say we can refactor if/when the nav is needed elsewhere. We really don't know if this new concept will fly, so probably better to wait on componetizing.